web/src/main/java/ca/hendriks/bartender/web/ingredient/IngredientsRestController.java

```diff
@@ -2,13 +2,13 @@
 
 import ca.hendriks.bartender.drinks.ingredient.Ingredient;
 import ca.hendriks.bartender.drinks.ingredient.IngredientRepository;
+import ca.hendriks.bartender.drinks.ingredient.IngredientType;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 @RestController
-@RequestMapping("/ingredients")
```

Comment:
Why remove this? This is the correct (base) path for all the mappings in this class.

Reply:
Reverted it back.

I like having custom URIs for the different operations